Doc/library/socket.rst

```diff
@@ -669,6 +669,30 @@ The following functions all create :ref:`socket objects <socket-objects>`.
    .. versionadded:: 3.3
 
 
+.. function:: socket.send_fds(sock, buffers, fds[, flags[, address]])
+
+   :func:`socket.send_fds` sends the list of file descriptors *fds*
+   over an :const:`AF_UNIX` socket, on systems which support the
+   :const:`SCM_RIGHTS` mechanism.
+
+   .. availability:: most Unix platforms, possibly others.
+
+   .. versionadded:: 3.8
+
+
+.. function:: socket.recv_fds(sock, bufsize, maxfds[, flags])
```

Comment:
What is bufsize? How should I use it? What is flags? Maybe suggest to read recvmsg() for the documentation of flags?